src/pal/src/configure.cmake

```diff
@@ -12,6 +12,9 @@ if(CMAKE_SYSTEM_NAME STREQUAL FreeBSD)
 elseif(NOT CMAKE_SYSTEM_NAME STREQUAL Darwin)
   set(CMAKE_REQUIRED_DEFINITIONS "-D_DEFAULT_SOURCE -D_POSIX_C_SOURCE=200809L")
 endif()
+
+list(APPEND CMAKE_REQUIRED_DEFINITIONS -D_FILE_OFFSET_BITS=64)
```

Comment:
This is actually just the definition for the configure script, but this is the default behaviour on 64 bit operating systems, and we require it on 32 bit operating systems to support large files.